app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -147,10 +149,35 @@ const classNameStory = (storyType, estimate) => classname (
   }
 );
 
-const StoryItem = ({ title, storyType, estimate, labels, state, ownedByInitials, ownedByName }) => (
+const StoryItem = ({
+  title,
+  storyType,
+  estimate,
+  labels,
+  state,
+  description,
+  ownedByInitials,
+  ownedByName,
+  requestedByName,
+  createdAt,
+  notes
```

Comment:
I noticed you've been using trailing comma in parts of the code. Is that the standard? If so, add it here :nail_care:

Reply:
We don't have a standard for that yet 😢, we'll fix it in a separate PR that will enforce better lint rules 😄